src/styles/index.css

```diff
@@ -0,0 +1,111 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  padding: 32px 19px 40px;
+}
```

Comment:
The padding shorthand property could be used here instead of individual padding properties to make the code cleaner. Consider combining `padding-top`, `padding-right`, `padding-bottom`, and `padding-left` into a single shorthand property.